changelog/unreleased/new-icons

```diff
@@ -0,0 +1,7 @@
+Enhancement: Add new icons
+
+We've added a `key` and `checklist` icons.
+
+https://fontawesome.com/icons/key?style=solid
+https://fontawesome.com/icons/tasks?style=solid
+https://github.com/owncloud/owncloud-design-system/pull/975
```

Comment:
The first link should always be the primary link. Could you move the PR link to the top?

Reply:
Done